kent2cky/GeoSearch
# Ft add autocomplete

**What does the PR do?**

Add autocomplete functionality to the search textbox

**Description of Tasks to be completed**

Have the page fetch suggestions for autocomplete from a remote server
using the fetch api.
Display the autocomplete suggestions under the textbox so user can select one
by `clicking` on them or by using `tab` and `enter` keys

**How should this be manually tested?**

After cloning the repo, cd into it and run npm install
After that, run the command gulp and it should be able to open up age page
on your web browser.
Your internet connection should be on.
Type in some keys into the search textbox on the page and 
the autocomplete suggestions should appear.
Clicking on any of the suggestions should be able to insert that suggestion 
into the textbox.
The `esc` key should be able to clear the suggestions.

**Any background context you want to provide?**
nil

**What are the relevant pivotal tracker stories?**

#3
